### DISCUSSION TITLE
[error][ERROR] 2FA token not provided

### DISCUSSION THREAD

**Discussion by abysso2** [dani-garcia/vaultwarden#3081] [Q&A]
* vaultwarden version: 1.27.0

* Install method: docker

* Clients used: web vault

* Reverse proxy and version: nginx, configuration from vaultwarden wiki

* MySQL/MariaDB or PostgreSQL version: splite3

* Other relevant details: webauthn as 2fa (yubikeys 5)

### Steps to reproduce
Login via webauthn as 2fa

### Expected behaviour
Login without error message

### Actual behaviour
Successfuly logged in, but showing the error-message in /var/log/bitwarden.log

### Troubleshooting data

/var/log/bitwarden.log: 
2022-12-31 10:58:44.326][error][ERROR] 2FA token not provided

**Comment by BlackDex** [comment]
It works for me ok.
Double check your reverse proxy settings, it probably adds some headers or changes some which block some features. Also some reverse proxy capture errors and modify the response, which also causes issues.

The error message you see there is normal, that is the notification for the clients to ask for the mfa token.
